manifests/f/FilenCloud/FilenSync/3.0.4/FilenCloud.FilenSync.locale.en-US.yaml

```
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+
+PackageIdentifier: FilenCloud.FilenSync
+PackageVersion: 3.0.4
+PackageLocale: en-US
+Publisher: Filen Cloud Dienste UG
+PublisherUrl: https://filen.io
+PublisherSupportUrl: https://support.filen.io
+PrivacyUrl: https://filen.io/privacy
+Author: Filen Cloud Dienste UG
+PackageName: Filen Sync
+PackageUrl: https://filen.io/apps
+License: Modified Read Only License
+LicenseUrl: https://github.com/FilenCloudDienste/filen-desktop/blob/master/LICENSE.md
+Copyright: © 2021 Filen Cloud Dienste UG
+CopyrightUrl: https://github.com/FilenCloudDienste/filen-desktop/blob/master/LICENSE.md
+ShortDescription: Sync files and folders to all devices connected to your account.
+Tags:
+- filen-cloud
+- filen-desktop

```

```suggestion
- filen-desktop
Documentations:
  - DocumentLabel: CLI
    DocumentUrl: https://docs.filen.io/docs/cli
  - DocumentLabel: SDK
    DocumentUrl: https://docs.filen.io/docs/sdk
  - DocumentLabel: API
    DocumentUrl: https://docs.filen.io/docs/api
```

This's some of their documentation I found in an earlier search.